gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -246,12 +246,12 @@ private static long getLong(Properties properties, String name, long dflt) {
     }
   }
 
-  private static boolean getBoolean(Properties properties, String name) {
-    return "true".equalsIgnoreCase(properties.getProperty(name, ""));
-  }
-
   private static boolean getBoolean(Properties properties, String name, boolean dflt) {
-    return "true".equalsIgnoreCase(properties.getProperty(name, dflt ? "true" : "false"));
+    if (dflt) {
```

Comment:
This implementation is making me go cross eyed trying to figure out the logic. What was wrong with the old implementation?